app.R

```diff
@@ -0,0 +1,843 @@
+            ),
+          p("Which sex has better survival rates overall? Which country has
+            better overall survival rates? Do your answers change when you look
+            at country and sex simultaneously?"),
+          fluidRow(
+            column(
+              width = 4,
+              offset = 0,
+              wellPanel(
+                checkboxGroupInput(
+                  inputId = "survivalSelection",
+                  label = "Select the sexes and countries you wish to compare",
+                  choices = c(
+                    "China-Females" = "China-Females",
+                    "China-Males" = "China-Males",
+                    "United Kingdom-Females" = "UK-Females",
+                    "United Kingdom-Males" = "UK-Males",
+                    "United States-Females" = "US-Females",
+                    "United States-Males" = "US-Males"
+                  )
+                )
+              )
+            ),
+            column(
+              width = 8,
+              offset = 0,
+              plotOutput("survivalChart1")
+            )
+          )
+        ),
+        #### Static Pyramid Page----
+        tabItem(
+          tabName = "staticPry",
+          h2("Static Population Pyramids"),
+          p("Population pyramids are common visualizations for life data. They
+            are composed of two histograms, breaking a countries population into
+            two sub-collections, typically by sex."),
+          tabsetPanel(
+            id = "cohortPyramid",
+            type = "tabs",
+            ##### Country to Country ----
+            tabPanel(
+              title = "Country to Country",
+              br(),
+              p("Compare each country's population pryamid to the other countries.
+                What do you notice?"),
+              fluidRow(
+                column(
+                  width = 4,
+                  offset = 0,
+                  wellPanel(
+                    radioButtons(
+                      inputId = "upperCountry",
+                      label = "Select the country for the upper graph",
+                      choices = c(
+                        "China",
+                        "United Kingdom",
+                        "United States"
+                      ),
+                      selected = "China"
+                    ),
+                    radioButtons(
+                      inputId = "lowerCountry",
+                      label = "Select the country for the lower graph",
+                      choices = c(
+                        "China",
+                        "United Kingdom",
+                        "United States"
+                      ),
+                      selected = "United Kingdom"
+                    )
+                  )
+                ),
+                column(
+                  width = 8,
+                  offset = 0,
+                  plotOutput("upperPyramid"),
```

Comment:
I think a br() should be inserted here because at first glance, Percentage alive looks like the title for the graph below. Also, I don't know if this is doable, but is there a way to have the graphs be displayed side by side? I usually find that it is easier to compare graphs that are side by side. In this case it is a bit harder especially because the student has to keep scrolling up and down to compare.

Reply:
I'm putting a space to see how that functions. However, I'm noticing that the legend is currently putting in all six pairings. When I was working yesterday, only the two pairings used in each graph were showing. I'm not sure what triggered the change.

Reply:
Looks better! And as for the legends, when I pull the code now it still only shows the two that are in each pyramid plot.